analyses/1.Transfer_Sepsis_to_TB.ipynb

```diff
@@ -0,0 +1,425 @@
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Initialize arguments to use in training the models\n",
+    "map_file = '../data/sample_classifications.pkl'\n",
+    "gene_file = '../data/intersection_genes.csv'\n",
+    "num_epochs = 1500\n",
+    "loss_scaling_factor = 1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Select a classifier architecture\n",
+    "label_to_encoding = {'sepsis': 1, 'healthy': 0}\n",
+    "sample_to_label = util.parse_map_file(map_file)\n",
+    "\n",
+    "net = model.ThreeLayerNet"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Split train and test data\n",
+    "\n",
+    "data_dirs = util.get_data_dirs('../data')                                                      \n",
+    "train_dirs, test_dirs = util.extract_test_dirs(data_dirs, 'tb', sample_to_label)                                               \n",
+    "\n",
+    "train_dirs, tune_dirs = util.train_tune_split(train_dirs, 2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "139e8fa8a1bd4621b8891c8741e66223",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "HBox(children=(IntProgress(value=0, max=1500), HTML(value='')))"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "erm_results = classifier.train_with_erm(net, map_file, train_dirs, tune_dirs, gene_file, num_epochs, \n",
+    "                                        label_to_encoding, device, logger, '../logs/sepsis_erm.pkl', 20)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "2b55be89fa8b42239830d330c5776bd8",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "HBox(children=(IntProgress(value=0, max=1500), HTML(value='')))"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "irm_results = classifier.train_with_irm(net, map_file, train_dirs, tune_dirs, gene_file, num_epochs, \n",
+    "                                        loss_scaling_factor, label_to_encoding, device, logger, '../logs/sepsis_irm.pkl', 100)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Print tuning set accuracy at the end of training for ERM and IRM models"
```

Comment:
Interesting. Did you expect IRM to have lower accuracy on the tuning set here?

Reply:
I wasn't sure what to expect, because I'm still not sure where the models fall on the overfitting/underfitting scale. IRM acts as a form of regularization, so I guess it makes sense that the performance would be worse since the models are only working with a few thousand samples.